docs/pages/enroll-resources/workload-identity/azure-oidc-federation.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: Configuring Workload Identity and Azure Federated Credentials
```

Comment:
Not super keen on the title atm, but it's consistent with the title for the AWS guides. Open to some suggestions here.